CHANGELOG.md

```diff
@@ -69,10 +69,11 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
 
 ### Ambasssador API Gateway + Ambassador Edge Stack
 
+- Feature: Mapping configuration now supports setting `auth_context_extentions` that allows setting the `check_settings` field in the per route configuration supported by `ext_authz` http filter.
 - Bugfix: Make Knative paths match on prefix instead of the entire path to better align to the Knative specification ([#3224]).
+- Change: The Helm chart has been moved into this repo, in the `charts/ambassador` directory.
 
 [#3224]: https://github.com/datawire/ambassador/issues/3224
```

Comment:
is this supposed to go with the knative entry in the changelog?

Reply:
It's a separate line item -- not sure I follow what you're getting at here?

Reply:
yeah... nevermind lol